src/main/resources/run-config.xml

```diff
@@ -0,0 +1,44 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="run" type="MavenRunConfiguration" factoryName="Maven">
+    <MavenSettings>
+      <option name="myGeneralSettings" />
+      <option name="myRunnerSettings">
+        <MavenRunnerSettings>
+          <option name="environmentProperties">
+            <map>
+              <entry key="INTEGRATION_ID" value="src/main/resources/integration-id.txt" />
```

Comment:
This was the location they where already in, but I have changed it.
You now have to make in the top directory a folder named `secrets` and put it there.
I couldn't think of a better name.